components/camera/rtppassthrough/rtppassthrough.go

```diff
@@ -1,18 +1,41 @@
-// Package rtppassthrough defines a source of RTP packets
+// Package rtppassthrough defines a Source of RTP packets
 package rtppassthrough
 
 import (
 	"context"
 
+	"github.com/google/uuid"
 	"github.com/pion/rtp"
 )
 
+// NilSubscription is the value of a nil Subscription.
+var NilSubscription = Subscription{ID: uuid.Nil}
+
+type (
+	// SubscriptionID is the ID of a Subscription.
+	SubscriptionID = uuid.UUID
+	// Subscription is the return value of a call to SubscribeRTP.
+	Subscription struct {
+		// ID is the ID of the Subscription
+		ID SubscriptionID
+		// Context will be cancelled when the RTP Subscription has terminated
+		// A successful call to Unsubscribe terminates the RTP Subscription with that ID
+		// An RTP Subscription may also terminate for other internal to the Source
+		// (IO errors, reconfiguration, etc)
+		context.Context
+	}
+)
+
 type (
 	// PacketCallback is the signature of the SubscribeRTP callback.
-	PacketCallback func(pkts []*rtp.Packet) error
+	PacketCallback func(pkts []*rtp.Packet)
```

Comment:
Previously, when `PacketCallback` (provided by the `SubscribeRTP` caller) returned an error, it would cause the subscription to enter a zombie state where RTP packets would no longer be processed, but the subscription would still be considered running. 

This was a partially implemented feature that didn't need to exist & was effectively a bug in the original design of the rtppassthrough.Source api.

I changed it to no longer be able to return an error, as there is already a way for the caller of `SubscribeRTP` to terminate a subscription (namely, `Unsubscribe`).